snow/consensus/avalanche/vertex.go

```diff
@@ -13,10 +13,6 @@ import (
 // Vertex is a collection of multiple transactions tied to other vertices
 type Vertex interface {
 	choices.Decidable
-	snowstorm.Whitelister
-
-	// Vertex verification should be performed before issuance.
-	Verify(context.Context) error
```

Comment:
nit: I assume we keep the vertex without Verfy since we don't really verify during bootstrap? If so please leave a little comment for people to come (or our selves in 6 months).

Reply:
added a comment